Makefile

```diff
@@ -85,14 +85,9 @@ fix-bom:
 verify-dep:
 	PASSES="dep" ./scripts/test.sh
 
-# TODO: https://github.com/etcd-io/etcd/issues/16610
-#
-# The golangci-lint doesn't verify sub modules. Before #16610 fixed, verify-lint
-# still depends on legacy {ineffassign,nakedret,unparam,...}_pass. These X_pass
-# will be removed when the golangci-lint covers all the sub modules.
 .PHONY: verify-lint
-verify-lint: verify-ineffassign
-	golangci-lint run --config tools/.golangci.yaml
+verify-lint:
+	PASSES="golangcilint" ./scripts/test.sh
```

Comment:
Could we rename the pass to just `lint`? It would make it consistent with `verify-lint`.